session/manager.go

```diff
@@ -102,7 +102,11 @@ func (manager *manager) Create(consumerID identity.Identity, proposalID int) (se
 }
 
 func (manager *manager) createSession(consumerID identity.Identity) (sessionInstance Session, err error) {
-	sessionInstance.ID = manager.generateID()
+	id, err := manager.generateID()
```

Comment:
Put directly to `sessionInstance.ID`, dont need temp variable here

Reply:
fixd